docs/docsite/rst/user_guide/playbooks_conditionals.rst

```diff
@@ -69,6 +69,7 @@ decide to do something conditionally based on success or failure::
 
 
 .. note:: both `success` and `succeeded` work (`fail`/`failed`, etc).
+.. warning:: Do NOT use `defined` or `default` to check if a variable was skipped. A skipped variable is still defined - metadata is inserted into the variable to record whether it was skipped, among other things.
```

Comment:
the variable is not skipped, the task is, the register reflects the task status.

Reply:
FYI this is already noted here `docs/docsite/rst/user_guide/playbooks_variables.rst`

Reply:
> the variable is not skipped, the task is, the register reflects the task status.

I changed the wording, does it look good now?

> FYI this is already noted here docs/docsite/rst/user_guide/playbooks_variables.rst

Good point. It was added in 4 years ago and people are still running into issues, so I think it deserves another mention.

Reply:
i would change this to make a ref to that note,  not that i expect most to read either and still hit this issue.

Reply:
Done. Look good?

Reply:
@bcoca Not sure if you saw my earlier message - I added a ref to the note, is it good to merge now?

Reply:
@caleb15 The docs team will review and merge.